### ISSUE TITLE
"Cannot read properties of null (reading 'number')" error from `@salesforce-ux/stylelint-plugin-slds`

### ISSUE THREAD

**Issue by SCWells72** [salesforce-ux/slds-linter#255]
This was reported by one of my users, but it looks like it's just a bug in `@salesforce-ux/stylelint-plugin-slds`. When first evaluating the following file CSS file:

```css
.myRule {
    font-weight: bold;
    display: inline-block;
}
```

the following error is raised:

```
Error when trying to validate file:///path/to/project/force-app/main/default/lwc/test/test.css TypeError: Cannot read properties of null (reading 'number')
    at /path/to/project/force-app/main/default/lwc/test/test.css:2:5
    at getStylingHooksForDensityValue (file:///path/to/project/node_modules/@salesforce-ux/stylelint-plugin-slds/build/utils/styling-hook-utils.js:11:60)
    at handleDensityPropForNode (file:///path/to/project/node_modules/@salesforce-ux/stylelint-plugin-slds/build/rules/no-hardcoded-value/handlers/densityHandler.js:24:24)
    at handleFontProps (file:///path/to/project/node_modules/@salesforce-ux/stylelint-plugin-slds/build/rules/no-hardcoded-value/handlers/fontHandler.js:26:7)
    at file:///path/to/project/node_modules/@salesforce-ux/stylelint-plugin-slds/build/rules/no-hardcoded-value/noHardcodedValue.rule.js:55:11
    at /path/to/project/node_modules/postcss/lib/container.js:363:18
    at /path/to/project/node_modules/postcss/lib/container.js:315:18
    at Rule.each (/path/to/project/node_modules/postcss/lib/container.js:63:16)
    at Rule.walk (/path/to/project/node_modules/postcss/lib/container.js:312:17)
    at /path/to/project/node_modules/postcss/lib/container.js:320:24
    at Root.each (/path/to/project/node_modules/postcss/lib/container.js:63:16) {
  postcssNode: Declaration {
    raws: { before: '\n    ', between: ': ' },
    type: 'decl',
    parent: Rule {
      raws: [Object],
      type: 'rule',
      nodes: [Array],
      parent: [Root],
      source: [Object],
      selector: '.partyHat',
      lastEach: 17,
      indexes: [Object],
      [Symbol(isClean)]: false,
      [Symbol(my)]: true
    },
    source: { input: [Input], start: [Object], end: [Object] },
    prop: 'font-weight',
    value: 'bold',
    [Symbol(isClean)]: false,
    [Symbol(my)]: true
  }
}
```

This is with 0.5.1 installed, by the way. Please let me know if I can provide any other diagnostic information.

**Comment by sf-nithappu** [comment]
@SCWells72  Thank you for reporting the issue. This issue is now fixed in `0.5.2` version.
